src/libtomahawk/utils/NetworkReply.cpp

```diff
@@ -90,7 +90,14 @@ NetworkReply::metaDataChanged()
             else
             {
                 disconnectReplySignals();
-                load( redir.toUrl() );
+                if(redir.toUrl().isRelative())
```

Comment:
Style: if ( redir.toUrl().isRelative() )

Wanna commit that quickly and I'll merge? ;-)